back-end/src/dms/tests/test_class_material_endpoints.py

```diff
@@ -0,0 +1,164 @@
+import json
+
+import pytest
+from django.test.client import MULTIPART_CONTENT, BOUNDARY, encode_multipart
+from dms.models.class_materials import ClassMaterial
+from dms.models.documents import File
+
+
+def assert_files_equal(file_data, file_data_original):
+    file = File.objects.get(id=file_data["content"].rsplit("/", maxsplit=1)
+                            [-1].rsplit("_", maxsplit=1)[0])
+
+    assert file.name == file_data_original["name"]
```

Comment:
Why is the signature so asymmetric, even though the arguments are called similar? Is it possible to make the arguments more the same?